### PR TITLE
fix: fix errors and typos

### PR DESCRIPTION
**Pull request by apapi**

### CODE REVIEW

**Comment by heathermiller** [comment]
Thanks for your contribution. While I can't really comment on the Chinese fixes, "hash trie" in the English documentation is correct. Our HashMaps are backed by an efficient trie implementation... Could you drop  the commits that rename Hash Tries? I assume the Chinese bits are correct :)

**Comment by SethTisue** [comment]
@hepin1989 interested in helping review Chinese-language PRs like these? or know anybody who is?

File: overviews/collections/concrete-immutable-collection-classes.md [hunk 1]
```diff
@@ -146,11 +146,11 @@ If you want to create a range that is exclusive of its upper limit, then use the
 
 Ranges are represented in constant space, because they can be defined by just three numbers: their start, their end, and the stepping value. Because of this representation, most operations on ranges are extremely fast.
 
-## Hash Tries
+## Hash Trees
```
Comment (hepin1989) [inline overviews/collections/concrete-immutable-collection-classes.md]: not change

File: zh-cn/overviews/collections/concrete-immutable-collection-classes.md [hunk 2]
```diff
@@ -42,7 +42,8 @@ List通常被认为是Scala中最重要的数据结构，所以我们在此不
     fibs: scala.collection.immutable.Stream[Int] = Stream(1, ?)
     scala> fibs.toList
     res9: List[Int] = List(1, 1, 2, 3, 5, 8, 13)
-    Vector（向量）
+
+## Vector（向量）
```
Comment (hepin1989) [inline zh-cn/overviews/collections/concrete-immutable-collection-classes.md]: better do not translate this

File: zh-cn/overviews/collections/concrete-immutable-collection-classes.md [hunk 3]
```diff
@@ -76,7 +77,8 @@ Vector结构通常被表示成具有高分支因子的树（树或者图的分
 
     scala> collection.immutable.IndexedSeq(1, 2, 3)
     res2: scala.collection.immutable.IndexedSeq[Int] = Vector(1, 2, 3)
-    Immutable stacks（不可变栈）
+
+## Immutable stacks（不可变栈）
```
Comment (hepin1989) [inline zh-cn/overviews/collections/concrete-immutable-collection-classes.md]: better do not translate this

File: zh-cn/overviews/collections/equality.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ language: zh-cn
 ---
 
 
-容器库有标准的等价性和散列法。首先，这个想法是为了将容器划分为集合，序列。不同范畴的容器总是不相等的。例如，即使包含相同的元素，`Set(1, 2, 3)` 与 `List(1, 2, 3)` 不等价。另一方面，在同一范畴下的容器是相等的，当且仅当它们具有相同的元素（对于序列：元素要相同，顺序要相同）。例如`List(1, 2, 3) == Vector(1, 2, 3)`， `HashSet(1, 2) == TreeSet(2, 1)`。
+容器库有标准的等价性和散列法。这个想法的第一步是将容器划分为集合，映射和序列。不同范畴的容器总是不相等的。例如，即使包含相同的元素，`Set(1, 2, 3)` 与 `List(1, 2, 3)` 不等价。另一方面，在同一范畴下的容器是相等的，当且仅当它们具有相同的元素（对于序列：元素要相同，顺序要相同）。例如`List(1, 2, 3) == Vector(1, 2, 3)`， `HashSet(1, 2) == TreeSet(2, 1)`。
```
Comment (hepin1989) [inline zh-cn/overviews/collections/equality.md]: uniform ->统一的
equality -> 计算相等性
hashing -> 计算hash值
idea -> 思想
first -> 首先
 and more...

**Comment by hepin1989** [comment]
I have to say ,this translation is more like translated by mechine not by a human.
@apapi 中文翻译讲究信达雅，如果真的要维护官方文档的翻译，我觉得至少得达到上面的要求吧。

@SethTisue I have to say ,this translation is far form it's quality.

**Comment by hepin1989** [comment]
I really don't know there are Chinese translation,but I have to say ,thr current translation is not that good,and have some wrong translated.I want to improve that ,but just hard to find time.

**Comment by SethTisue** [comment]
we'd certainly welcome additional fixes. every little bit helps.

**Comment by heathermiller** [comment]
Given that both the Chinese and English improvements have issues, @SethTisue do you advise that we close? Or give @apapi a chance to remedy these issues?

**Comment by SethTisue** [comment]
let's wait for @apapi.

also, I think @hepin1989 comments about the overall quality are about the translation as a whole, rather than about the quality of @apapi's edits. we could still accept @apapi's edits (if revised to reflect the feedback so far) as long as they are some kind of improvement.

**Comment by apapi** [comment]
@heathermiller thank you for educating me on HashMap implementation.
@hepin1989 I agree with you the overall quality of Chinese translation is poor. I just want to fix the type setting and obvious errors in these parts I have read.

File: zh-cn/overviews/collections/concrete-immutable-collection-classes.md (hunk 2 above)
Comment (apapi) [inline zh-cn/overviews/collections/concrete-immutable-collection-classes.md]: this commit is about the type setting

File: zh-cn/overviews/collections/concrete-immutable-collection-classes.md (hunk 3 above)
Comment (apapi) [inline zh-cn/overviews/collections/concrete-immutable-collection-classes.md]: this commit is about the type setting

**Comment by SethTisue** [comment]
thanks all!

**Pull request merged**